### PR TITLE
[KYUUBI #1563] Fix broken link and add new link in `CONTRIBUTION.md`

### PR DESCRIPTION
**Pull request by ulysses-you**
### _Why are the changes needed?_

closes https://github.com/apache/incubator-kyuubi/issues/1563

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -51,12 +51,10 @@ You can make various types of contributions to Kyuubi, including the following b
 - Improve test coverage - [![codecov](https://codecov.io/gh/apache/incubator-kyuubi/branch/master/graph/badge.svg)](https://codecov.io/gh/apache/incubator-kyuubi)
 - Report bugs and better help developers to reproduce
 - Review changes
-- Make a pull request
+- Make a pull request follow [guideline](https://kyuubi.apache.org/pull_request.html)
```
Comment (yaooqinn) [inline CONTRIBUTING.md:54]:
```suggestion
- [Make a pull request](https://kyuubi.apache.org/pull_request.html)
```

**Review by SteNicholas — APPROVED** [review]
LGTM.

**Comment by yaooqinn** [comment]
thanks, merged to master

**Comment by ulysses-you** [comment]
thank you for review @yaooqinn @SteNicholas

**Pull request closed without merging**